apps/src/templates/CheckboxDropdown.jsx

```diff
@@ -19,17 +19,19 @@ const CheckboxDropdown = ({
     </button>
     <ul className="dropdown-menu">
       <form>
-        {allOptions.map(option => (
-          <li key={`${name}-${option}`} className="checkbox form-group">
+        {Object.keys(allOptions).map(optionKey => (
+          <li key={`${name}-${optionKey}`} className="checkbox form-group">
```

Comment:
you shouldn't need the name here -- the key only needs to be unique with respect to the element's siblings in the DOM, so `optionKey` should be enough.